### PR TITLE
Adding oraclecloudapps.com from Oracle Autonomous Database

### PR DESCRIPTION
**Pull request by pmandala**
Public Suffix List (PSL) Pull Request (PR) Template
====

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

__Submitter affirms the following:__ 

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The submitter acknowledges that it is their responsibility to maintain the domains within their section. This includes removing names which are no longer used, retaining the _psl DNS entry, responding to e-mails to the supplied address. Failure to maintain entries may result in removal of individual entries or the entire section.

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====

Oracle Corporation is an American multinational computer technology company headquartered in Austin, Texas. In 2020, Oracle was the third-largest software company in the world by revenue and market capitalization.

Organization Website: https://www.oracle.com/
https://www.oracle.com/autonomous-database/

Reason for PSL Inclusion
====

Oraclecloudapps,com, oraclegovcloudapps.com and oraclegovcloudapps.uk are for customer use when using the Oracle Cloud Infrastructure platform for Automonous Database. These domains are used for customers accessing oracle application deployed in against their database. 

Customer are reported Cookie Tossing issues hence we are fixing to deliver Customer web content must only be accessible via domains in the Public DNS Suffix List. 

Number of users this request is being made to serve: In millions

DNS Verification via dig
=======

```
dig +short TXT _psl.oraclecloudapps.com
"https://github.com/publicsuffix/list/pull/2109"
```

```
 dig +short TXT _psl.oraclegovcloudapps.com
"https://github.com/publicsuffix/list/pull/2109"
```

```
dig +short TXT _psl.oraclegovcloudapps.uk 
"https://github.com/publicsuffix/list/pull/2109"
```

Results of Syntax Checker (`make test`)
=========

```
========================================================================
Testsuite summary for libpsl 0.21.5
========================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
========================================================================
```

### CODE REVIEW

**Comment by wdhdev** [comment]
* [ ]  Expiration (Note: Must STAY >2y at all times)     
  * [ ] oraclecloudapps.com expires 2025-06-17
  * [ ] oraclegovcloudapps.com expires 2025-08-08
  * [ ] oraclegovcloudapps.uk expires 2024-10-09
* [ ]  DNS _psl entries (Note: Must STAY in place)
  * [ ] `_psl.oraclecloudapps.com`
  * [ ] `_psl.oraclegovcloudapps.com`
  * [ ] `_psl.oraclegovcloudapps.uk`
* [x]  Tests pass
* [x]  Sorting
* [ ]  Reasoning/Organization description
* [x]  Non-personal email address

All domains in this PR are not renewed for >2y, also all of the `_psl` TXT records are missing.

You also state you are looking to get around Cloudflare and Letsencrypt limitations, can you please state why this is. Also you do not actually provide the reason as to why these domains should be included in the PSL, you only provide what these domains are used for. Please state reasons for the inclusion, e.g. cookie separation, URL highlighting, etc.

**Comment by pmandala** [comment]
1. Working on the extending expiration of domain- Need some time here.
2. We are unable to make dig work. Could you please help how to debug on that?
3. Updated Reasoning in the main PR template 
4. There is a typo regarding Cloudflare limitation. We are not looking around it now. Fixed the template. Sorry for the confusion

**Comment by wdhdev** [comment]
> Working on the extending expiration of domain- Need some time here.

That's alright, please note this won't be approved until then.

> We are unable to make dig work. Could you please help how to debug on that?

Not to be rude, but do you how DNS works? You need to place a `_psl` TXT record on each domain with this content: `https://github.com/publicsuffix/list/pull/2109`

This is used to verify that you control the domains you are requesting. You must work for Oracle in order to have this approved.

**Comment by pmandala** [comment]
Fixed DNS records

**Comment by wdhdev** [comment]
Can confirm they are in place. We just need the domain names renewed for >2y then we can likely get this approved. 😊

**Comment by pmandala** [comment]
We fixed the expiration date of all domains.
oraclecloudapps.com expires 2033-08-07
 oraclegovcloudapps.com expires 2033-08-07
 oraclegovcloudapps.uk expires 2033-10-08

**Comment by wdhdev** [comment]
@simon-friedberger This PR is ready to be merged. I confirmed the domain expiry dates via a WHOIS lookup.

**Comment by dnsguru** [comment]
Running unit tests

**Comment by dnsguru** [comment]
Have we confirmed submitter necus with Oracle?

**Comment by dnsguru** [comment]
Have we confirmed submitter necus with Oracle?


Why is this separate from other Dyn entries?

**Comment by pmandala** [comment]
[Oracle Cloud Autonomous Database Service ](https://www.oracle.com/autonomous-database/) serves few web applications which uses those domain names to access over public internet

**Comment by groundcat** [comment]
[patch](https://patch-diff.githubusercontent.com/raw/publicsuffix/list/pull/2109.patch) of this PR shows:

```
From 9eb019968ee708e099618bbcf69a9a4821630bba Mon Sep 17 00:00:00 2001
From: Phaneendhar Mandala <phaneendhar.mandala@oracle.com>
Date: Tue, 20 Aug 2024 10:51:02 -0700
Subject: [PATCH] Adding oraclecloudapps.com from Oracle Autonomous Database
```

Observed the use of `oraclecloudapps.com` at https://docs.oracle.com/en/cloud/paas/autonomous-database/serverless/adbsb/apex-vanity-url.html

Observed the use of `oraclegovcloudapps.com` and `oraclegovcloudapps.uk` at https://blogs.oracle.com/machinelearning/post/announcing-new-urls-for-oracle-machine-learning-notebooks-and-restful-services

_psl TXT records are in place.

**Comment by dnsguru** [comment]
That seems reasonable as "connective tissue" for evidence.  We on
confirming submitter.

We likely want to get these integrated with dyn entries vs a new section
like this

**Comment by simon-friedberger** [comment]
@pmandala Can you please move these to the section labeled `// Customer OCI - Oracle Dyn https://cloud.oracle.com/home https://dyn.com/dns/` as requested by @dnsguru ?

File: public_suffix_list.dat [hunk 1]
```diff
@@ -12158,6 +12158,12 @@ translated.page
 // Submitted by Jacob Lee <jacob@autocode.com>
 autocode.dev
 
+// Autonomous Database OCI - Oracle Dyn https://cloud.oracle.com/home https://dyn.com/dns/
```
Comment (simon-friedberger) [inline public_suffix_list.dat:12161]: Should be added here
 https://github.com/publicsuffix/list/pull/2109/files#diff-b4fc013fa0dff30019626bef00fba7a39210a002b463194b258b8b8ae302215bL12527

**Review by simon-friedberger — CHANGES_REQUESTED** [review]

File: public_suffix_list.dat (hunk 1 above)
Comment (pmandala) [reply public_suffix_list.dat]: Resolved the order recommendation and moved after Customer-OCI

**Comment by pmandala** [comment]
Rebased the the branch after new check in.

Submitting the tests again
 ```============================================================================
Testsuite summary for libpsl 0.21.5
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================

**Comment by pmandala** [comment]
How do I reopen the this pull request?

**Comment by pmandala** [comment]
Reopen the pull request after rebase

**Comment by simon-friedberger** [comment]
You deleted the branch that it was trying to get added. Maybe you can reopen if you recreate that branch.

**Comment by pmandala** [comment]
I already recreated that branch with the [same name adbs](https://github.com/pmandala/list/tree/adbs), still the `reopen and commit` is disabled

**Comment by pmandala** [comment]
Reopen the branch

**Comment by pmandala** [comment]
Reopen the branch

**Comment by pmandala** [comment]
Reopen

**Comment by pmandala** [comment]
reopen

**Comment by pmandala** [comment]
reopen

**Comment by pmandala** [comment]
Could some one please help in reopening this PR?

**Comment by pmandala** [comment]
reopen

**Comment by wdhdev** [comment]
Just create a new commit and open a new PR referencing this one.

**Comment by pmandala** [comment]
New PR created https://github.com/publicsuffix/list/pull/2130

**Pull request closed without merging**